.circleci/config.yml

```diff
@@ -24,6 +24,23 @@ jobs:
         paths:
         - venv/
         key: python-packages-v1-{{ .Branch }}-{{ checksum "requirements.txt" }}-{{ checksum "constraints.txt" }}
+  verify-generated-sql:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Verify that all of generated SQL is committed
+          command: |
+            sudo apt-get update
+            python3.7 -m venv venv/
```

Comment:
Within a docker container, I don't think there's much benefit to creating a virtual environment.

Reply:
Outside of a virtual environment there is an issue with permissions when doing `pip install`:

`ERROR: Could not install packages due to an EnvironmentError: [Errno 13] Permission denied: '/usr/local/lib/python3.7/site-packages/apipkg'`

Reply:
That's surprising. I may not understand conventions here as well as I thought I did. If the venv is solving a problem, keeping it sounds entirely reasonable.